### DISCUSSION TITLE
feat: add a (kick) started admin template

### DISCUSSION THREAD

**Discussion by tomkuijsten** [microsoft/fluentui-blazor#369] [Ideas]
# 🙋 Feature Request

A started blazor project with an "admin template", like this:

https://github.com/siminture/fluentui-starter

He even has a nice sample online:

https://simmy.tech/fluentui-starter/login

## 🤔 Expected Behavior

A git repo or project template to get kickstarted with fluent UI on blazor server.

## 😯 Current Behavior

This repo contains an example folder, but that's not really useful in this case (just a demo website for the components).

## 💁 Possible Solution

I would expect at least a wireframe with things like header, user profile, navigation menu, some sample pages (dashboard, login, purchase orders etc)

## 🔦 Context

No real alternatives found, the above mentioned git repo is what I'm looking for, but that one is not for blazor and not updated for over 2 years.

## 💻 Examples

See possible solutions

**Comment by vnbaaij** [comment]
Hi @tomkuijsten,

The template you mentioned is a nice starting point indeed, I think it can be (mostly) replicated with the components from the Incubation Lab. An example of that can be found here: https://brave-cliff-0c0c93310.azurestaticapps.net/Lab/Overview

For the library we need to focus on the components themselves. We do not have the time and/or resources to work on templates unfortunately. Besides, I think templates are very much influenced by taste and preferences so it will be very hard to 'do the right thing'. 

Perhaps a repo can be set up by you and others that focuses on this? From the library site we would be happy to support such an effort with help and guidance.

I'll turn this issue into a discussion so others can chime in.

**Comment by tomkuijsten** [comment]
Thanks @vnbaaij, and I understand that your focus is on the components itself and not a template. Your suggestion of a new repo is the right way forward. However, it's not my intention to create it myself, I asked the question because the UI is not our core feature nor our specialism. We are willing to buy a framework but want it to be based upon Fluent UI and blazor server so we can continue working on it with our c# knowledge.

**Comment by ericmqt** [comment]
Just wanted to chime in here since this discussion seems relevant:

Something a little more light-weight like MudBlazor's [wireframes](https://mudblazor.com/getting-started/wireframes) would be welcome as well. 

The fluentui-blazor templates, while a nice starting point, aren't really "complete", for lack of a better word. I recognize that this is partly because of some "labs" components that haven't been completed (NavMenu being a big one). 

Ideally, the fluentui-blazor templates would generate something similar in layout to the Office 365 Admin web app (https://admin.microsoft.com/). A basic nav bar, header with an application name and some buttons, and a content area. To me, this is _exactly_ what comes to mind when I think "start a new web app with Fluent UI".

It looks like the polished-up versions of the labs components in the new-components branch are making their way into the library, so that's great news towards this end, I just don't want to have to mess around with a bunch of CSS while trying to get a new project off the ground.

**Reply by calloncampbell** [reply]
I'd love to see the following.

**Comment by EwoutdBoer** [comment]
I think some good example pages / components would realy help the project. A client of mine, just turned down usage of the use of the Fluentui-blazor components and went for an html template from theme-forest. Mainly while they don't get a notion of what it would look like in the end.
I like what they created for shadcn/ui. They have several implementations for their components, like the card component. They also have example pages and blocks of functionality. What is really useful is the 'lift' functionality in the blocks section. There you could lift the sections on the page and copy only that section.
I think having these samples would be a good 'selling' point for both end-users and developers. It will also speed-up development when you could copy, paste, alter examples
I would like to help out with this
